### ISSUE TITLE
Bottom Navigation Bar - Webfonts as icon

### ISSUE THREAD

**Issue by csimpi** [nativescript-community/ui-material-components#322]
How can I add font-awesome icons to the tabs?
I saw this as a workaround, but this feels nuts, more like avoiding the issue than fixing it:

https://github.com/nativescript-community/ui-material-components/issues/225#issuecomment-848049332

Is there any better approach? This package should handle this out of the box on template level.

**Comment by farfromrefug** [comment]
@csimpi it works the same way as in any N component and is explain on N website and in the issue you linked. The Vue demo app works well so it should work with angular too. If there is an issue might be angular related. I will live this open for now for you but i wont fix any angular issue.

**Comment by csimpi** [comment]
@farfromrefug Wrong, it doesn't work the same way as documented. Why I would've asked about this ??? By joke or what? 
Don't be rude, and please click to the link and see not I'm the only one who couldn't use it. The workaround is not a solution it's a BAD practice that should be corrected. 
This UI package and the documentation is terrible.

**Comment by farfromrefug** [comment]
@csimpi ok i am not the rude one here. If you dont like that COMMUNITY handled package, then dont use it...
If you think there is an issue why not try to replicate it in a core sample project? Then i could fix it.

Finally thank you for your nice comment, seeing your great contributions to OSS i really appreciate it.

**Comment by farfromrefug** [comment]
@fpaaske thanks a lot for pointing this out. I will fix the example!
